media-proxy/src/rdma_session.c

```diff
@@ -79,16 +79,24 @@ int rx_rdma_shm_init(rx_rdma_session_context_t *rx_ctx, memif_ops_t *memif_ops)
     rx_ctx->memif_conn_args.socket = rx_ctx->memif_socket;
     rx_ctx->memif_conn_args.interface_id = memif_ops->interface_id;
     rx_ctx->memif_conn_args.buffer_size = (uint32_t)rx_ctx->transfer_size;
-    rx_ctx->memif_conn_args.log2_ring_size = 4;
+    rx_ctx->memif_conn_args.log2_ring_size = 2;
```

Comment:
Reducing the ring buffer size from 16 to 4 entries? Serio?

Reply:
Yes that was the intention. I have not performed any tests, it was just my guess that 16 is more then it is needed. 
Maybe I will leave this number unchanged for know. In future a tuning based on tests needs to be performed.

Reply:
I think in the future we'll have to create a configuration path to let the cluster administrator to adjust that based on metrics.